query07.sql

```diff
@@ -2,5 +2,15 @@
   How many trips started on one day and ended in the next?
 */
 
--- Enter your SQL query here
-select ...
+-- For 2019:
+
+SELECT 
+	COUNT(trip_id)
+FROM indego_trips_2019_q2 
+WHERE EXTRACT(DAY from start_time) + 1 = EXTRACT(DAY from end_time)
```

Comment:
Right track, but slightly under-counting. To really use this method, you would have to check for month boundaries as well (e.g. April 30 - May 1). The solution I used is [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L16-L25) (or a more compact equivalent solution [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L40-L42))